packages/client/src/components/FilteringArea/FilteringArea.css

```diff
@@ -9,43 +9,76 @@
   display: flex;
 }
 
+.filtering-checkbox-mobile-view-for-category,
+.filtering-checkbox-mobile-view-for-participants,
+.filtering-checkbox-mobile-view-for-timeframe {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+  max-width: 35.899rem;
+  gap: 0.626rem;
+  flex-wrap: wrap;
+}
+
+.filtering-checkbox-element {
+  flex-grow: 1;
+  font-size: 1.125rem;
+}
+
 .filtering-area {
-  width: 838px;
-  gap: 0 1rem;
+  width: 50rem;
+  gap: 0.518rem 1rem;
   justify-content: center;
   margin-left: auto;
   margin-right: auto;
+  flex-direction: column;
+}
+
+.filtering-title p {
+  font-size: 1.25rem;
+  line-height: 1.25rem;
+  font-weight: 400;
+  /* margin-top: 1.25rem; */
 }
 
 .filtering-category {
-  flex-direction: column;
-  width: 264px;
-  gap: 0.2rem 0;
+  flex-direction: row;
+  /* width: 264px; */
+  gap: 1.155rem;
+  height: 2.406rem;
+  justify-content: center;
 }
 
-.filtering-category-title {
+/* .filtering-category-title {
```

Comment:
Hi, could we remove all the spare lines?

Reply:
Yes, it's done